lib/telemetry_metrics.ex

```diff
@@ -84,12 +84,10 @@ defmodule Membrane.TelemetryMetrics do
     end
   end
 
-  defp emit_event?(event) do
-    cond do
-      not @enabled -> false
-      @events == :all -> true
-      is_list(@events) -> event in @events
-      true -> false
-    end
+  cond do
+    not @enabled -> defp emit_event?(_event), do: false
+    @events == :all -> defp emit_event?(_event), do: true
+    is_list(@events) -> defp emit_event?(event), do: event in @events
+    true -> defp emit_event?(_event), do: false
```

Comment:
I've told you to check for warnings :stuck_out_tongue: 
When `MIX_ENV=test` it's fine (thus CI passes) but in dev when `@enabled` is false, there's a warning about unused module attribute.

I've found a way to solve it - use variables for code executed at compile time:
```suggestion
  enabled = Application.compile_env(:membrane_telemetry_metrics, :enabled, false)
  events = Application.compile_env(:membrane_telemetry_metrics, :events, :all)

  cond do
    not enabled -> defp emit_event?(_event), do: false
    events == :all -> defp emit_event?(_event), do: true
    is_list(events) -> defp emit_event?(event), do: event in events
    true -> defp emit_event?(_event), do: false
```